schema.sql

```diff
@@ -4,8 +4,9 @@ CREATE TABLE animals (
     id serial PRIMARY KEY,
     name varchar(100) not null,
     date_of_birth date not null,
-    escape_attempts int,
-    neutered boolean,
-    weight_kg decimal not null
+    escape_attempts int not null,
+    neutered boolean not null,
+    weight_kg decimal not null,
+    species varchar(100) not null
 );
 
```

Comment:
Great job @mahabubx7 🤝

- [ ] As stated in the project requirements, I would recommend you add the command that adds the `species` column to your `animals` table. Add this to the `schema.sql` file.
- `ALTER TABLE animals ADD species varchar(255);`